packages/why3-base/why3-base.0.88.3/opam

```diff
@@ -0,0 +1,66 @@
+opam-version: "1.2"
+maintainer: "guillaume.melquiond@inria.fr"
+authors: [
+  "François Bobot"
+  "Jean-Christophe Filliâtre"
+  "Claude Marché"
+  "Guillaume Melquiond"
+  "Andrei Paskevich"
+]
+
+homepage: "http://why3.lri.fr/"
+license: "GNU Lesser General Public License version 2.1"
+doc: "http://why3.lri.fr/doc/"
+bug-reports: "https://gitlab.inria.fr/why3/why3/issues"
+dev-repo: "https://gitlab.inria.fr/why3/why3.git"
+
+tags: [
+  "deductive"
+  "program verification"
+  "formal specification"
+  "automated theorem prover"
+  "interactive theorem prover"
+]
+available: [ ocaml-version >= "4.02.3" ]
+
+# Jessie3 (frama-c plugin) is *disabled* because it is not ready
+build: [
+  ["./configure" "--prefix" prefix "--disable-frama-c"
+                 "--disable-ide" { !conf-gtksourceview:installed }]
+  [make "-j%{jobs}%" "all" "opt" "byte"]
+]
+
+install: [make "install" "install-lib"]
+
+remove: [
+  ["rm" "%{bin}%/why3"]
+  ["rm" "-r" "%{lib}%/why3"]
+  ["rm" "-r" "%{share}%/why3"]
+]
+
+build-doc: [
+  [make "doc" "stdlibdoc" "apidoc"]
+  [make "DOCDIR=%{doc}%/why3" "install-doc"]
+]
+
+depends: [
+  "ocamlfind"
+  "menhir"
+  "num"
+]
+
+depopts: [
+  "lablgtk"
+  "conf-gtksourceview"
+  "zarith"
+  "camlzip"
+  "ocamlgraph"
+  "coq"
+]
+
+conflicts: [
+  "lablgtk" {< "2.14.2"}
+  "ocamlgraph" {< "1.8.2"}
+  "coq" {< "8.4"}
+  "coq" {>= "8.8"}
```

Comment:
The two lines above can be squeezed into one, as follow:
```"coq" {< "8.4" & >= "8.8"}```

Reply:
> The two lines above can be squeezed into one, as follow: `"coq" {< "8.4" & >= "8.8"}`

Are you sure about that? Conflicting against both Coq versions less than 8.4 and Coq versions more than 8.8 is not the same as conflicting against Coq versions that are both less than 8.4 and more than 8.8 (which there are none, obviously).

Reply:
mmh, right, I'm not sure. Let's keep that, sorry.

Reply:
And just to be thorough, I tried with a disjunction (which has the intended semantics), but `opam` then complains
```
lablgtk < 2.14.2 & ocamlgraph < 1.8.2 & coq (< 8.4 | >= 8.8) is not a valid conjunction
```

Not sure if this is a bug or a feature.